### DISCUSSION TITLE
Laser GRBL stops between Passes

### DISCUSSION THREAD

**Discussion by seinereiner** [arkypita/LaserGRBL#1662] [General]
Hi,

following issue, I'm using multiple passes for Cuttin with my Neje Master 2S max. Sometimes, Laser GBL stops after last line of one (mostly first, but some time other passes) and did nor start next pass. Last commands are M5 and S0.
Then I can stop, restart job by option restart some line before last commend, and job continues with next passes, mostly for all upcoming passes.

Does anybody have an idea?

BR

**Comment by arkypita** [comment]
Could you tell me if, on the last command, you see the green tick or the clock?

![image](https://user-images.githubusercontent.com/8782035/143834080-0c5a8b11-bde2-4856-8459-5ba10f89033f.png)

**Reply by seinereiner** [reply]
Hi,

yes, is a little clock...

**Comment by arkypita** [comment]
Each command LaserGRBL send should be acknowledge by the machine with an "ok".
Clock means "command sent" and green tick mean "ok received".

At runtime there may be more than one command with the clock because a small buffer is managed by the controller.

If at the end of job a command remains with the clock it is because one of these "ok" has been lost, not necessarily the one of the last command sent, but it could also be an earlier one.

The reasons why these "ok" are lost are mostly attributable to electrical communication disturbances.

read more here: https://github.com/arkypita/LaserGRBL/issues/1598

**Comment by arkypita** [comment]
What version of LaserGRBL do you use?

**Comment by seinereiner** [comment]
Thanks for this information arkypita.
The provided thread was really interresting. Same on my site is, I also added a second machine (Neje Master 2s Max) to the Laptop, where the Atomstack A5 was running fine for a long time witout any issue, still not yet. The issue only appears with the second one. I will try to use settings suggested in other thread.
My actual version is 4.6.0.
